### PR TITLE
Add e2e ovn-local-gateway presubmit jobs for 4.8+

### PR DESCRIPTION
**Pull request by jluhrsen**
Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml [hunk 1]
```diff
@@ -141,6 +141,70 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (abhat) [inline ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml:145]: do we want these to always run or only when we think we are making changes that might break local gateway. On second thoughts, figuring that out may be tricky and so better to run these always?

Comment (jluhrsen) [reply ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml:145]: yeah, there is a way to run CI based on which files/folders are touched. at least I read about it, but I haven't worked on anything with it yet and it only makes sense if some jobs are clearly never relevant unless some specific set of files are touched. I'm not sure if that's the case here.

Comment (abhat) [reply ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml:145]: OK, I think we can optimize further later.

**Pull request merged**